packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -12,22 +12,23 @@ import {
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 
-export const StoryPromoLi = styled.li.attrs({
+export const StoryPromoLiBase = styled.li.attrs({
   role: 'listitem',
 })`
-
-  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-    border-bottom: 0.0625rem solid ${C_LUNAR};
-  }
-
   ${({ border }) =>
-    !border &&
+    border &&
     `
-      @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-          border-bottom: none;
-      }
-    `}
+    @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+      border-bottom: 0.0625rem solid ${C_LUNAR};
+    }
+  `}
 
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    border-bottom: none;
+  }
```

Comment:
Is this style is setting `border-bottom: none;` on an element that doesn't have a `border-bottom` value set?

Reply:
@simonsinclair the `border-bottom:none;` is a default for when `border` is `false` and `border-bottom: 0.0625rem solid ${C_LUNAR};` is set for when `border` is `true` at the same break point.

Reply:
`border-bottom: none;` should already be the default without this style - unless something is adding a border to it elsewhere?

Reply:
Could this media query be removed if l.21 was changed to ${GEL_GROUP_2_SCREEN_WIDTH_MAX} ?

Reply:
@Simon2828 I just confirmed [from here](https://bbc.github.io/psammead/?path=/story/components-storypromo-storypromolist--default) that the `border-bottom:none;` isn't needed anymore, it was previously when we were negating the logic [see comment](https://github.com/bbc/psammead/pull/3588#discussion_r452306854).

Reply:
@simonsinclair fixed.